csrc/codebase/mmdet/object_detection.cpp

```diff
@@ -12,16 +12,78 @@ namespace mmdeploy::mmdet {
 
 ResizeBBox::ResizeBBox(const Value& cfg) : MMDetection(cfg) {
   if (cfg.contains("params")) {
-    score_thr_ = cfg["params"].value("score_thr", 0.f);
+#ifdef EVAL_MODE
+    INFO("EVAL_MODE, SDK uses postprocessing score_thr for evaluating performance.");
+    if (cfg["params"].contains("conf_thr")) {
+      // for mobilev2yolov3
+      score_thr_ = cfg["params"].value("conf_thr", 0.f);
+    } else {
+      score_thr_ = cfg["params"].value("score_thr", 0.f);
+    }
+#else
+    INFO("inference mode, SDK uses fixed 0.3 score_thr_ for inferencing.");
+    score_thr_ = 0.3;
+#endif
     min_bbox_size_ = cfg["params"].value("min_bbox_size", 0.f);
   }
 }
+std::vector<Tensor> ResizeBBox::GetDetsLabels(const Value& prep_res, const Value& infer_res) {
+  std::vector<Tensor> results;
+  if (infer_res.contains("dets") && infer_res.contains("labels")) {
+    results.push_back(infer_res["dets"].get<Tensor>());
+    results.push_back(infer_res["labels"].get<Tensor>());
+    return results;
+  } else if (infer_res.contains("detection_output") && (!infer_res.contains("dets")) &&
+             (!infer_res.contains("labels"))) {
+    int img_width = prep_res["img_metas"]["img_shape"][2].get<int>();
+    int img_height = prep_res["img_metas"]["img_shape"][1].get<int>();
+    auto detection_output = infer_res["detection_output"].get<Tensor>();
+    auto* detection_output_ptr = detection_output.data<float>();
+    // detection_output: (1, num_det, 6)
+    TensorDesc labeldesc = detection_output.desc();
+    int batch_size = detection_output.shape()[0];
+    int num_det = detection_output.shape()[1];
+    labeldesc.shape = {batch_size, num_det};
+    Tensor labels(labeldesc);
+    TensorDesc detdesc = detection_output.desc();
+    detdesc.shape = {batch_size, num_det, 5};
+    Tensor dets(detdesc);
+    auto* dets_ptr = dets.data<float>();
+    auto* labels_ptr = labels.data<float>();
+    for (int c = 0; c < batch_size; ++c) {
```

Comment:
The nested loop scared me. Can we do some thing to optimize the loop? such as add `__restrict__` to all these ptrs?